src/test/java/de/hpi/bpt/argos/eventProcessing/EventReceiverTest.java

```diff
@@ -73,14 +78,23 @@ public void testReceiveEvent() {
 		assertEquals(HttpStatusCodes.SUCCESS, request.getResponseCode());
 
 		List<Event> events = PersistenceAdapterImpl.getInstance().getEventsOfEventType(testEventType.getId());
+		assertEquals(true, events.size() >= 1);
 
-		assertEquals(1, events.size());
+		assertEquals(oldStatus, getEntityStatus());
+
+		List<String> webSocketMessages = webSocket.awaitMessages(1, 1000);
+		assertEquals(1, webSocketMessages.size());
+		ArgosTestUtil.assertWebSocketMessage(webSocketMessages.get(0), PersistenceArtifactUpdateType.CREATE, "Event");
 	}
 
 	@Test
 	public void testReceiveEvent_InvalidEventTypeId_NotFound() {
+		String oldStatus = getEntityStatus();
+		testMapping.setTargetStatus("");
```

Comment:
This would be more meaningful if you actually had a target status that is still not reached. (in the test befor you already proved, that an empty status doesnt change the status)